### ISSUE TITLE
Mapping a field to a Dimension without creating explicitly a table

### ISSUE THREAD

**Issue by danielmittag** [zazuko/cube-creator#441]
Let's start with a concrete example of UDB0037, focussing on the two columns **Lärmbelastete Einheit** and **Periode**:

Here's the data (spaces have been added for readability):

```
Referenzjahr,GeoEinheit,Laermart,      Lärmbelastete Einheit,Beurteilung,Periode,Angabe,Wert,Einheit
2015,        Schweiz,   Eisenbahnlaerm,Bewohner am Wohnort,  >IGWLSV,    Tag,    Anteil,0.20,%
2015,        Schweiz,   Eisenbahnlaerm,Wohnungen,            >IGWLSV,    Tag,    Anteil,0.19,%
2015,        Schweiz,   Eisenbahnlaerm,Gebaeude,             >IGWLSV,    Tag,    Anteil,0.09,%
2015,        Schweiz,   Eisenbahnlaerm,Bewohner am Wohnort,  >IGWLSV,    Nacht,  Anteil,1.05,%
2015,        Schweiz,   Eisenbahnlaerm,Wohnungen,            >IGWLSV,    Nacht,  Anteil,1.04,%
...
```
Both are dimensions coming from a single column. For **Lärmbelastete Einheit** I mapped it as a "simple property", but for **Periode** I created a dimension table (same CSV, same column, the fact that I mapped the same column twice for DE and EN doesn't matter here)

![GitHubIssue_441a](https://user-images.githubusercontent.com/17119538/105456702-e8bc6a80-5c85-11eb-9ff5-bc1a4a92f010.png)

When I take a look at the output of the first observation on https://test.lindas.admin.ch/sparql/# it looks like this:

DESCRIBE <https://environment.ld.admin.ch/foen/UDB0037/2/observation/2015/Schweiz/Eisenbahnlaerm/Wohnungen/%3EIGWLSV/Tag/Anteil/0.19/%25>

![GitHubIssue_441b](https://user-images.githubusercontent.com/17119538/105457033-77c98280-5c86-11eb-909b-acca94df39e8.png)

In other words: **Periode** is (as expected) mapped as a **dimension**, while **Lärmbelastete Einheit** is mapped as a **literal attribute**.

The trouble is, that most (if not all) users will use the simple mapping method of **Lärmbelastete Einheit** while imho the mapping as a dimension like in **Periode** is semantically correct. I would therefore suggest some sort of shortcut like a "Map as dimension" checkbox that tells the pipeline to map the column as a dimension.

Remarks:
- I tried to inspect the dimension metadata, but on https://test.lindas.admin.ch/sparql/# my query DESCRIBE `<https://environment.ld.admin.ch/foen/UDB0037/2/shape/>` didn't return anything
- Maybe the "Map as dimension" feature belongs rather to Cube Designer than to CSV Mapper

**Comment by l00mi** [comment]
@danielmittag Sorry, but what is this issue about? What is not working and what do you expect to work?

**Comment by danielmittag** [comment]
@l00mi I hadn't finished writing when you commented. Now it's complete.

**Comment by l00mi** [comment]
Sorry for my too early comment.

This is a valid point. We might create such a shortcut. The only problem i currently see is how to deal with different languages with this behaviour.

Definitely we need to explain in the user documentation why we should create a new table by definition for "categorical" values.

**Comment by l00mi** [comment]
Documented.
